raster/src/main/scala/geotrellis/raster/io/geotiff/reader/TiffTagsReader.scala

```diff
@@ -106,7 +113,7 @@ object TiffTagsReader {
         byteBuffer.readDoublesTag(tiffTags, tagMetadata)
     }
 
-  implicit class ByteBufferTagReaderWrapper(val byteBuffer: ByteBuffer) extends AnyVal {
+  implicit class ByteReaderTagReaderWrapper(val byteBuffer: ByteReader) extends AnyVal {
```

Comment:
No idea, that was written a pretty long time ago, I think. If I had to guess, it's because of the wide range of types that need to accessed and evaluated to read in TiffTags.